dependencies/CMakeLists.txt

```diff
@@ -62,6 +62,10 @@ if (SIMDJSON_COMPETITION)
   add_library(competition-ujson4c ujson4c/src/ujdecode.c)
   target_include_directories(competition-ujson4c PUBLIC ujson4c/3rdparty ujson4c/src)
 
+  initialize_submodule(cxxopts)
```

Comment:
This shouldn't be inside if(SIMDJSON_COMPETITION) which is OFF by default on MSVC. I believe that is the reason AppVeyor fails.

Reply:
Damn it. I was coming here to write this. You stole my thunder.

@yoannlr  You can reproduce the problem anywhere by doing `cmake .. -DSIMD_COMPETITION=OFF`. So it is not visual studio per se that is the problem.

Reply:
Thanks, I did put it outside of SIMDJSON_COMPETITION, it worked with `cmake .. -DSIMD_COMPETITION=OFF`.